extensions/vscode/src/constants.ts

```diff
@@ -107,7 +103,6 @@ export const Contexts = {
 export const enum Views {
   Project = "posit.publisher.project",
   HomeView = "posit.publisher.homeView",
-  Credentials = "posit.publisher.credentials",
   ContentRecords = "posit.publisher.contentRecords",
```

Comment:
We should clean this one up, my quick search indicates its not used anymore.

```suggestion
```

Reply:
I'll do this in a separate PR 👍

Reply:
https://github.com/posit-dev/publisher/pull/2164 opened up to resolve this.